dist/Client.js

```diff
@@ -324,21 +356,48 @@ class APIClient {
     }
   }
   /**
-     * @async
-     * @function listDetectors
-     * @summary Get the list of available custom detectors
-     * @description Lists the metadata of all the custom detectors
-     * owned by the API user, thus ready to preditc with on rasters
-     * @returns {Promise<[Object]>} A JSON list of the available detectors
-     * @throws {APIError} Containing error code and text
-     */
+    * @summary Delete a raster, identified by an UUID
+    * @description Delete a given raster, identified by an UUID and
+    * owned by the API user
+    * @param {String} rasterId UUID of the raster
+    * @returns {Promise<Boolean>} Whether or not removal was successful
+    * @throws {APIError} Containing error code and text
+    */
+
+
+  async deleteRasterById(rasterId) {
+    if (!uuidValidator(rasterId)) {
+      throw new ValidationError('Invalid UUID string: ', rasterId);
+    } // Send HTTP request
+
+
+    const response = await this._request('/rasters/'.concat(rasterId, '/'), 'DELETE'); // Error management
+
+    if (!response.ok) {
+      const text = await response.text();
+      throw new APIError('Error deleting raster with status '.concat(response.status), text);
+    } // Return all went good
+
+
+    return true;
+  }
+  /**
+   * @async
+   * @function listDetectors
+   * @summary Get the list of available custom detectors
+   * @description Lists the metadata of all the custom detectors
+   * owned by the API user, thus ready to preditc with on rasters
+   * @returns {Promise<[Object]>} A JSON list of the available detectors
+   * @throws {APIError} Containing error code and text
+   */
 
 
   async listDetectors() {
     const response = await this._request('/detectors/');
 
     if (!response.ok) {
-      throw new APIError(`Error getting detectors list with status ${response.status}`);
+      const text = await response.text();
+      throw new APIError(`Error getting detectors list with status ${response.status}`, text);
```

Comment:
Shouldn't we send the status is we say its a status ?

I feel we could unify this error handling into a common function `checkResponse(response)`
Because we always do the same thing. I'm not sure having the error message of which operation went wrong ('Error getting detectors list..') is actually useful, because its part of the stack trace anyway (you'll see that it failed in get detector). So I feel we could just throw `APIError` with the status code and response text and then up to the client to print nice messages.